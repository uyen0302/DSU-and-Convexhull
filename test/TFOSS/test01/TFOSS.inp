5

4
9 4
3 15
10 -4
0 -6

1
1 2

3
0 10
0 2
1 6

10
2 4
1 -2
4 -10
18 6
0 -1
8 -7
13 10
8 17
5 6
2 3

4
15 8
1 -9
0 -9
3 -10
