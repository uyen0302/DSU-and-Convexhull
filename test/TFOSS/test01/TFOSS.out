450
0
64
745
514
